deployment config for nodejs wrapper

farskipper opened the pull request:
We'll need a npm user to publish this. Maybe this one https://www.npmjs.com/~hyperledger-ci ?

Then store the auth token (npmrc) in the `npm_credentials` file. Similar to `pypi_credentials`

vimmerru commented:
@farskipper Could you fix conflicts? Also as i understand we need integrate the same changes as for CI PR.

vimmerru commented:
@farskipper We can create new npm user, but i am a bit afraid that this user can post artifacts with the same name. What do you think about this?

farskipper commented:
@vimmerru Ok, I updated the branch and fixed the merge conflict.

Not sure what you meant by "afraid this user can post artifacts with the same name."  If we create a new npm user, I'll give it permission to publish to `npmjs.com/package/indy-sdk`; and this `Jenkinsfile.cd` is setup to publish pre-releases as well as full releases.

ArthurManz commented:
Why is this PR paused?

farskipper commented:
I can transfer ownership of the npm package `indy-sdk` to `~hyperledger-ci` or whatever user you'll use to publish the builds. Just let me know which user should own it.
